test/03-worker.test.js

```diff
@@ -17,7 +17,7 @@ test('worker pipe', async function ({ is, plan, comment, teardown }) {
   const messages = []
   const response = new Promise((resolve) => {
     pipe.on('data', (data) => {
-      messages.push(data.toString())
+      messages.push(JSON.parse(data.toString()).value)
```

Comment:
As mentioned above, lets create new fixture so we dont have to change this test yet.

Reply:
updated